front/components/actions/mcp/CreateMCPServerModal.tsx

```diff
@@ -94,6 +94,7 @@ export function CreateMCPServerModal({
           await createMCPServerConnection({
             connectionId: cRes.value.connection_id,
             mcpServerId: createServerRes.server.id,
+            provider: authorization.provider,
```

Comment:
passing provider when creating mcp connection so that we can fetch and check the connection in the endpoint